webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -247,6 +257,12 @@ const RepositorySetsTab = () => {
       simpleContentAccess, apiSortParams, statusSelected, STATUS_LABEL],
   );
 
+  useEffect(() => {
+    if (orgId && orgStatus !== STATUS.RESOLVED) {
```

Comment:
I think this logic probably needs to be different in order to guarantee the `getOrganization` call is made when needed.

Reply:
noted. Going to address it here -> https://projects.theforeman.org/issues/35604 do you think i should remove the useffects and get organization bits?